tests/DummyServerLib/DummyServer.cpp

```diff
@@ -193,6 +193,46 @@ void DummyServer::mainLoop() {
     else if(data == "ACC? AXIS2") {
       _serialPort->send("AXIS_2=" + std::to_string(acc[1]));
     }
+
+    else if(data.find("HEX ") == 0) {
+      auto tokens = tokenise(data);
+      if(tokens.size() <= 3) {
+        _serialPort->send("12345 Syntax error: HEX needs axis and value");
+      }
```

Comment:
You picked an unnecessarily complicated syntax for the hex command. Just positional, fixed number arguments would be sufficient. The rest is tested already, so we can concentrate on the hex part of it.
We should also test the mixed writing (last argument must not have 0x extension)
```HEX 0x12bab 0xd0d010 acbdef```
```
HEX?
0x12bab 0xd0d010 acbdef
```